frontend/src/pages/admin/project-edit.js

```diff
@@ -329,42 +329,44 @@ const ProjectEdit = () => {
                   </HelperText>
                 </FormHelperText>
               </FormGroup>
-              <FormGroup fieldId="owner" label="Owner">
```

Comment:
I would prefer that we keep the FormGroup, and set `isDisabled` for the toggleOwner memo.

https://v5-archive.patternfly.org/components/menus/menu-toggle#menutoggle

Paired with updating the `HelperTextItem` content, you can explain that owner will be assigned to the current user and can be updated after saving.